packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -357,19 +357,21 @@ const diffusersImg2ImgDefaultPrompt = "Turn this cat into a dog";
 const diffusersVideoDefaultPrompt = "A man with short gray hair plays a red electric guitar.";
 
 const diffusers_default = (model: ModelData) => [
-	`from diffusers import DiffusionPipeline
+	`import torch
+from diffusers import DiffusionPipeline
 
-pipe = DiffusionPipeline.from_pretrained("${model.id}")
+pipe = DiffusionPipeline.from_pretrained("${model.id}", dtype=torch.bfloat16, device_map="cuda")
```

Comment:
instead of "cuda" just do "auto" everywhere?

Reply:
I had a talk with @sayakpaul and @linoytsaban 
Diffusers does not have `"auto"` yet, so we would need to specify "cuda" everywhere.

Reply:
The other alternative is using `.to("cuda")` as being done elsewhere. But `device_map="cuda"` is faster than that and hence the suggestion. If for consistency reasons, we want to do `to("cuda")`, that is okay, too.

Reply:
I could add `accelerate` here: https://github.com/huggingface/huggingface.js/blob/b85a0f0c91247cf434e7129377890d11782ef9e0/packages/tasks/src/model-libraries-snippets.ts#L351